tests/setup.py

```diff
@@ -14,6 +14,7 @@
     include_package_data=True,
     install_requires=[
         'ducktape@git+https://github.com/redpanda-data/ducktape.git@fd01d72b8c82ae852986bb288c69818c7ffb4de8',
+        'python-keycloak@git+https://github.com/oleiman/python-keycloak.git@451695d81656e0b8a7f3c279283fa664cc762dbd',
```

Comment:
chore: fork this into Redpanda's org (you should be able to click "fork" on the upstream repo and then from the "Owner" drop-down select "redpanda-data".  Let either IT or Ivo know if you can't do it).

Reply:
> chore: fork this into Redpanda's org (you should be able to click "fork" on the upstream repo and then from the "Owner" drop-down select "redpanda-data". Let either IT or Ivo know if you can't do it).

Would this still be necessary when https://github.com/redpanda-data/redpanda/pull/13116 lands?

Reply:
> > chore: fork this into Redpanda's org (you should be able to click "fork" on the upstream repo and then from the "Owner" drop-down select "redpanda-data". Let either IT or Ivo know if you can't do it).
> 
> Would this still be necessary when #13116 lands?

Missing some context - will this include python-keycloak as part of ducktape?

If so.... then I'm not sure... maybe that's an @ivotron question.  I guess it's a race then to see who lands first?

Reply:
#13116 has merged, so I've updated `tests/setup.py` to reference the mainline `python-keycloak` lib.